packages/synapse-react-client/src/lib/utils/hooks/useOverlay.tsx

```diff
@@ -78,6 +78,7 @@ export function useOverlay(
   const OverlayComponent = useCallback(
     () => (
       <Popover
+        TransitionComponent={Fade}
```

Comment:
For some reason, the default `Grow` transition component would trigger an error if this Popover opened in a Dialog.